### PR TITLE
Feature/#754 multi tab support for view editing

### PR DESCRIPTION
**Pull request by mateuszhyski**

### CODE REVIEW

File: WebContent/WEB-INF/jsp/viewEdit.jsp [hunk 1]
```diff
@@ -385,7 +385,20 @@
...
+                    <input type="hidden" id="id" name="view.id" value="${status.value}"/>
+                  </td>
+                </tr>
+              </spring:bind>
 
+                <tr>
+                  <td class="formField" width="250">
+                    <input type="hidden" id="dwrScriptSessionid" name="dwrScriptSessionid" value="">
+                  </td>
+                  <td class="formError">${status.errorMessage}</td>
```
Comment (grzesiekb) [inline WebContent/WEB-INF/jsp/viewEdit.jsp]: probably it is not needed "<td class="formError">${status.errorMessage}</td>"

File: src/com/serotonin/mango/Common.java [hunk 2]
```diff
@@ -26,12 +26,7 @@
 import java.security.NoSuchAlgorithmException;
 import java.text.MessageFormat;
 import java.text.ParseException;
-import java.util.ArrayList;
-import java.util.Enumeration;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Locale;
-import java.util.ResourceBundle;
+import java.util.*;
```
Comment (grzesiekb) [inline src/com/serotonin/mango/Common.java]: Please keep previous imports with the explicit declaration.

File: src/com/serotonin/mango/Common.java [hunk 3]
```diff
@@ -41,8 +36,7 @@
 import org.apache.commons.httpclient.auth.AuthScope;
 import org.apache.commons.httpclient.params.HttpClientParams;
 import org.apache.commons.httpclient.params.HttpConnectionManagerParams;
-import org.directwebremoting.WebContext;
-import org.directwebremoting.WebContextFactory;
+import org.directwebremoting.*;
```
Comment (grzesiekb) [inline src/com/serotonin/mango/Common.java]: Please keep previous imports with the explicit declaration.

File: src/com/serotonin/mango/ScriptSessionAndUsers.java [hunk 4]
```diff
@@ -0,0 +1,77 @@
+package com.serotonin.mango;
+
+import com.rits.cloning.Cloner;
+import com.serotonin.mango.vo.User;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.directwebremoting.*;
```
Comment (grzesiekb) [inline src/com/serotonin/mango/ScriptSessionAndUsers.java]: Please use imports with an explicit declaration.

File: WebContent/WEB-INF/jsp/viewEdit.jsp (hunk 1 above)
Comment (mateuszhyski) [reply WebContent/WEB-INF/jsp/viewEdit.jsp]: ok, thank you

File: src/com/serotonin/mango/Common.java (hunk 2 above)
Comment (mateuszhyski) [reply src/com/serotonin/mango/Common.java]: ok, thank you

File: src/com/serotonin/mango/Common.java (hunk 3 above)
Comment (mateuszhyski) [reply src/com/serotonin/mango/Common.java]: ok, thank you

File: src/com/serotonin/mango/ScriptSessionAndUsers.java (hunk 4 above)
Comment (mateuszhyski) [reply src/com/serotonin/mango/ScriptSessionAndUsers.java]: ok, thank you

**Pull request merged**